docs/eigenda/operator-guides/run-a-node/registration.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
```

Comment:
Can we change the ordering so that Registration comes first? Otherwise, this will likely lead to errors when the user attempts to run their node for the first time.

Reply:
We actually want registration to come second. This is now the advised flow. cc @shrimalmadhur

Reply:
yea registration should happen after node is up. it's fine to see some log messages in the node that node is not registered.